src/include/edl/sgx_ecdsa.edl

```diff
@@ -22,10 +22,31 @@ enclave {
                                                 [out, size=quote_size] uint8_t *quote);
 
 		enclave_verifier_err_t ocall_ecdsa_verify_evidence([user_check] enclave_verifier_ctx_t *ctx,
-                                                                sgx_enclave_id_t enclave_id,
-                                                                [in, string] const char *name,
-								[in, size=quote_size] sgx_quote3_t *pquote,
-								uint32_t quote_size)
+															sgx_enclave_id_t enclave_id,
+															[in, string] const char *name,
```

Comment:
please format these code.

Reply:
The clang-format does not format the .edl file. I have manually formatted it to be consistent with the existing code.